File: java/trip-service-kata/src/main/java/org/craftedsw/tripservicekata/trip/TripDaoDelegate.java
package org.craftedsw.tripservicekata.trip;

import org.craftedsw.tripservicekata.user.User;

import java.util.List;

public class TripDaoDelegate {
    public TripDaoDelegate() {
    }

    List<Trip> findTripsByUser(User user) {
        return TripDAO.findTripsByUser(user);
    }
}

File: c#/TripServiceKata/TripServiceKata/Trip/TripDaoDelegate.cs
using System.Collections.Generic;

namespace TripServiceKata.Trip
{
    public class TripDaoDelegate : ITripDaoDelegate
    {
        public TripDaoDelegate()
        {
        }

        public List<Trip> FindTripsByUser(User.User user)
        {
            return TripDAO.FindTripsByUser(user);
        }
    }
}